examples/framesjs-starter/app/examples/simplified-api/[[...routes]]/route.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnyJson } from "../new-api";
+import {
+  FrameImage,
+  Frame,
+  FramesApp,
+  FrameLink,
+  renderFramesToResponse,
+} from "../react-api";
+
+type FrameState = {
+  clicked: number;
+};
+
+function isFrameState(
+  frameState: AnyJson | undefined
+): frameState is FrameState {
+  if (frameState && typeof frameState === "object" && "clcked" in frameState) {
+    return typeof frameState.clicked === "number";
+  }
+
+  return false;
+}
+
+export const framesApp = (
+  <FramesApp>
+    <Frame index>
+      {({ frameState }) => (
+        <>
+          <FrameImage>
+            <div>Test</div>
+          </FrameImage>
+          <FrameLink
+            path="/next"
+            label="Next"
+            state={{
+              clicked: isFrameState(frameState) ? frameState.clicked + 1 : 1,
```

Comment:
@davidfurlong @stephancill this is a next frame state (the frame state of the destination frame will be set to this value)